packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/__tests__/toolbar.spec.tsx

```diff
@@ -1,77 +1,73 @@
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import Toolbox from '../../toolbox';
 
-const mockDbotStore = {
-    run_panel: {
-        is_running: false,
-    },
-    save_modal: {
-        toggleSaveModal: jest.fn(),
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+jest.mock('@deriv/bot-skeleton/src/scratch/hooks/block_svg', () => jest.fn());
+
+window.Blockly = {
+    Colours: {
+        RootBlock: {},
     },
-    load_modal: {
-        toggleLoadModal: jest.fn(),
+    derivWorkspace: {
+        svgBlockCanvas_: {
+            getBoundingClientRect: jest.fn(),
+        },
+        options: {
+            readOnly: true,
+        },
     },
-    toolbar: {
-        has_redo_stack: false,
-        has_undo_stack: false,
-        closeResetDialog: jest.fn(),
-        onResetOkButtonClick: jest.fn(),
-        onResetClick: jest.fn(),
-        onSortClick: jest.fn(),
-        onUndoClick: jest.fn(),
-        onZoomInOutClick: jest.fn(),
-        is_dialog_open: false,
+    Xml: {
+        textToDom: () => ({
+            childNodes: [
+                {
+                    tagName: 'example',
+                },
+            ],
+        }),
     },
-    quick_strategy: {},
-    dashboard: {},
 };
+describe('Toolbox component', () => {
+    let wrapper: ({ children }: { children: JSX.Element }) => JSX.Element;
+    beforeAll(() => {
+        const mock_store = mockStore({});
+        const mock_DBot_store = mockDBotStore(mock_store, mock_ws);
 
-jest.mock('Stores/useDBotStore', () => ({
-    useDBotStore: jest.fn(() => mockDbotStore),
-}));
-
-jest.mock('@deriv/stores', () => ({
-    ...jest.requireActual('@deriv/stores'),
-    observer: jest.fn(x => x),
-    useStore: jest.fn(() => ({
-        ui: {
-            is_mobile: false,
-        },
-    })),
-}));
-
-describe('Toolbar component', () => {
-    beforeEach(() => {
-        (useDBotStore as jest.Mock).mockReturnValue(mockDbotStore);
+        wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock_store}>
+                <DBotStoreProvider ws={mock_ws} mock={mock_DBot_store}>
+                    {children}
+                </DBotStoreProvider>
+            </StoreProvider>
+        );
     });
-    it('should render Toolbar', () => {
-        render(<Toolbar />);
-        expect(screen.getByTestId('dashboard__toolbar')).toBeInTheDocument();
+    it('should render Toolbox with content wrapper is open', () => {
+        render(<Toolbox />, { wrapper });
+        expect(screen.getByTestId('dashboard__toolbox')).toBeInTheDocument();
+        expect(screen.getByTestId('db-toolbox__content-wrapper')).toHaveClass('db-toolbox__content-wrapper active');
     });
+    it('should render Toolbox with content wrapper is open', () => {
```

Comment:
change the name of the text, 2 tests have the same name: 'should render Toolbox with content wrapper is open'